docs/examples/camera_display.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Typo: gronud → ground</p><p> </p><p>Side note: TelescopeFrame also works for this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/ctapipe/pull/2266/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Actually, TelescopeFrame breaks the hillas ellipse overlay it seems (but that is fixed I think in #2141 ) 
```
AttributeError: 'HillasParametersContainer' object has no attribute 'x'
```